inference-engine/tests/functional/inference_engine/transformations/algebraic_simplification.cpp

```diff
@@ -106,14 +73,12 @@ TEST(algebraic_simplification, multiply_negative_tests)
 
     auto expected = ngraph::NodeVector{a, b, add_a_0_0, c, add_b_0_0};
     auto results = f->get_results();
-    for (size_t i = 0; i < results.size(); i++)
-    {
+    for (size_t i = 0; i < results.size(); i++) {
         ASSERT_EQ(expected.at(i), results.at(i)->input_value(0).get_node_shared_ptr());
     }
 }
 
-TEST(algebraic_simplification, multiply_prod_negative)
-{
+TEST(algebraic_simplification, multiply_prod_negative) {
```

Comment:
These tests use v0 ops... Maybe we should modify them.

Reply:
I plan to refactor this passes so I suggest to include test refactoring into this future effort.

Reply:
BTW this tests were disabled in ngraph. So now they at least working.